packages/server-wallet/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+// From the tslint.json we used previously
```

Comment:
I think you can strip this file down quite a bit. It might also be a good time to introduce some more opinionated rules like grouped and sorted imports.

Reply:
Done